app/helpers/locales_helper.rb

```diff
@@ -70,6 +70,12 @@ def language_options
     @language_options ||= language_options_for selectable_locales, nil, incomplete_locales
   end
 
+  def angular_locales
+    selectable_locales.map do |locale|
```

Comment:
Someday I think we may need to objectify locales.. I keep having this thought that all of this craziness around locales would be mitigated by just having a Locale object which would have key, name, fallback, etc.